CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+* Support pandas v2. [#1671] (@corneliusroemer and @victorlin)
```

Comment:
Reminder to rebase and update the Changelog so it doesn't get out-of-sync due to the latest release of v26.1.0.

Reply:
Or merge via CLI to fix it during the merge as explained in https://github.com/nextstrain/augur/issues/1620#issuecomment-2327972371